lyricsgenius/artist.py

```diff
@@ -52,9 +52,10 @@ def add_song(self, newsong, verbose=True):
             self._num_songs += 1
             return 0 # Success
         else:
-            if verbose:
-                print("Can't add song by {newsong.artist}, artist must be {self.name}.".format(newsong=newsong,self=self))
-            return 1 # Failure        
+            self._songs.append(newsong)
```

Comment:
I don't want to allow combinations of multiple artists. One solution could be creating a new `Album()` class that allows for combinations of multiple artists. But, that feels excessive. I'm not sure what the best solution is right now.